test/codegen/julia/config.jl

```diff
@@ -5,8 +5,8 @@ using Comonicon
 using Configurations
 
 @option struct OptionA
```

Comment:
can we have a separate test (in a separate test module) so we can have both tests (original and the new case)

Reply:
@Roger-luo , sure, I can move it to a separate files, but why? All the original tests are [still there](https://github.com/comonicon/Comonicon.jl/pull/229/files#diff-b0b97d6d43551d1c81128515a32a025f76026fa98bb961bca6fd7d2e739f1e80R33), I only replaced `@main` with `@cast` to allow multiple entry points for different tests.

Reply:
oh you are right, sry I misread the git diff of `OptionA`.